src/test/java/org/jabref/model/groups/TexGroupTest.java

```diff
@@ -28,10 +38,20 @@ public void containsReturnsTrueForEntryInAux() throws Exception {
     @Test
     public void containsReturnsTrueForEntryNotInAux() throws Exception {
         Path auxFile = Paths.get(TexGroupTest.class.getResource("paper.aux").toURI());
-        TexGroup group = new TexGroup("paper", GroupHierarchyType.INDEPENDENT, auxFile, new DefaultAuxParser(new BibDatabase()), new DummyFileUpdateMonitor());
+        TexGroup group = new TexGroup("paper", GroupHierarchyType.INDEPENDENT, auxFile, new DefaultAuxParser(new BibDatabase()), new DummyFileUpdateMonitor(), metaData);
         BibEntry notInAux = new BibEntry();
         notInAux.setCiteKey("NotInAux2017");
 
         assertFalse(group.contains(notInAux));
     }
+
+    @Test
+    public void ReturnsTrueIfPathRelative() throws Exception {
```

Comment:
test methods should usually reflect the expected outcome (and maybe gives some hints concering the setting/context). E.g. here `getFilePathReturnsRelativePath`